bower.json

```diff
@@ -54,7 +54,8 @@
     "paper-tooltip": "PolymerElements/paper-tooltip#^2.0.0",
     "platinum-sw": "PolymerElements/platinum-sw#^1.3.0",
     "polymer": "polymer/polymer#^2.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0"
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0",
+    "paper-toggle-button": "^2.0.0"
```

Comment:
I think we've sorted these alphabetically, so please add this at the correct location.